### PR TITLE
Add database models and migrations

### PR DESCRIPTION
**Pull request by hobogalaxy**

### CODE REVIEW

File: API/groups/models.py [hunk 1]
```diff
@@ -1,3 +1,67 @@
 from django.db import models
+from django.contrib.auth.models import User
 
-# Create your models here.
+
+class UserInfo(models.Model):
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
+    date_of_creation = models.DateField(auto_now_add=True)
```
Comment (urbrob) [inline API/groups/models.py]: How about change name to `created_at`

File: API/groups/models.py [hunk 2]
```diff
@@ -1,3 +1,67 @@
...
+    content = models.TextField(max_length=1000)
+
+    def __str__(self):
+    	return self.content
+
+
+class Answer(models.Model):
+    question = models.ForeignKey(Question, on_delete=models.CASCADE)
+    content = models.CharField(max_length=200)
+    true = models.BooleanField(default=False)
```
Comment (urbrob) [inline API/groups/models.py]: How about `correct`

File: API/groups/models.py [hunk 3]
```diff
@@ -1,3 +1,67 @@
...
+    MEMBER = 'member'
+    MODERATOR = 'moderator'
+    ROLES = (
+            (MEMBER, 'Member'),
+            (MODERATOR, 'Moderator')
+    )
+    role = models.CharField(max_length=10, choices=ROLES, default=MEMBER)
+
+
+class Categories(models.Model):
```
Comment (urbrob) [inline API/groups/models.py]: `Category`

**Comment by hobogalaxy** [comment]
All done

**Pull request merged**